### PR TITLE
Use smaller alpine in tests

### PR DESCRIPTION
**Pull request by brybacki**
Signed-off-by: Bartosz Rybacki <brybacki@redhat.com>

**What this PR does / why we need it**:

Change big 5Gig fedora image to smaller 1Gig alpine. 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

Just checking if tests are faster and more predictable. 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by brybacki** [comment]
/retest 
retesting just to see how stable/predictable it is

File: tests/utils.go [hunk 1]
```diff
@@ -31,13 +31,15 @@ const (
 	waitTime            = 600 * time.Second
 	veleroCLI           = "velero"
 	forceBindAnnotation = "cdi.kubevirt.io/storage.bind.immediate.requested"
+	//fedoraUrl           = "docker://quay.io/kubevirt/fedora-with-test-tooling-container-disk"
+	alpineUrl = "docker://quay.io/kubevirtci/alpine-with-test-tooling-container-disk:2205291325-d8fc489"
```
Comment (arnongilboa) [inline tests/utils.go:35]: Why a specific tag was added?

File: tests/utils.go [hunk 2]
```diff
@@ -518,8 +520,8 @@ func WaitVirtualMachineDeleted(client kubecli.KubevirtClient, namespace, name st
 }
 
 func NewDataVolumeForFedoraWithGuestAgentImage(dataVolumeName string, storageClass string) *cdiv1.DataVolume {
```
Comment (ShellyKa13) [inline tests/utils.go:522]: you should probably change the name of the function :)

File: tests/utils.go [hunk 3]
```diff
@@ -31,13 +31,15 @@ const (
 	waitTime            = 600 * time.Second
 	veleroCLI           = "velero"
 	forceBindAnnotation = "cdi.kubevirt.io/storage.bind.immediate.requested"
+	//fedoraUrl           = "docker://quay.io/kubevirt/fedora-with-test-tooling-container-disk"
+	alpineUrl = "docker://quay.io/kubevirtci/alpine-with-test-tooling-container-disk:2205291325-d8fc489"
 )
 
 func CreateVmWithGuestAgent(vmName string, storageClassName string) *kvv1.VirtualMachine {
```
Comment (ShellyKa13) [inline tests/utils.go:38]: does this alping image has a guest agent?

File: tests/utils.go [hunk 4]
```diff
@@ -31,13 +31,15 @@ const (
 	waitTime            = 600 * time.Second
 	veleroCLI           = "velero"
 	forceBindAnnotation = "cdi.kubevirt.io/storage.bind.immediate.requested"
+	//fedoraUrl           = "docker://quay.io/kubevirt/fedora-with-test-tooling-container-disk"
```
Comment (ShellyKa13) [inline tests/utils.go:34]: delete if not used?

File: tests/utils.go (hunk 1 above)
Comment (brybacki) [reply tests/utils.go:35]: there is no `latest` tag for that image
```
Using default tag: latest
Error response from daemon: manifest for quay.io/kubevirtci/alpine-with-test-tooling-container-disk:latest not found: manifest unknown: manifest unknown
```

File: tests/utils.go (hunk 2 above)
Comment (brybacki) [reply tests/utils.go:522]: ;) thanks, did not notice that

File: tests/utils.go (hunk 3 above)
Comment (brybacki) [reply tests/utils.go:38]: yes, that is why it worked here

File: tests/utils.go (hunk 2 above)
Comment (brybacki) [reply tests/utils.go:522]: done

File: tests/utils.go (hunk 4 above)
Comment (brybacki) [reply tests/utils.go:34]: will push after current tetsts pass

**Comment by brybacki** [comment]
I can see that tests are stable, passed twice already. 

I have pushed the cleanups to address review comments.

**Comment by brybacki** [comment]
one test failed, looking
/test pull-kvp-functional-test

**Comment by brybacki** [comment]
ok, this test should actually wait to VM to be deleted. it looks like a flaky test

**Pull request merged**